examples/ct/ctmapper/README.md

```diff
@@ -25,20 +25,24 @@ go build ./examples/ct/ctmapper/lookup
 go build ./cmd/createtree/
 tree_id=$(./createtree \
     --admin_server=localhost:8090 \
-    --signature_algorithm=ECDSA)
+    --hash_strategy TEST_MAP_HASHER \
```

Comment:
nit: be consistent with the other flags and use '='

Reply:
Done, well spotted.